__tests__/app.test.js

```diff
@@ -26,5 +26,42 @@ describe("/api/categories", () => {
           expect(typeof category.description).toBe("string");
         });
       });
-  });    
+  });
+});
+
+describe("/api/reviews/:review_id", () => {
+  test("GET - status:200 - return review object with id ", () => {
+    return request(app)
+      .get(`/api/reviews/1`)
+      .expect(200)
+      .then((response) => {
+        expect(typeof response.body.review).toEqual("object");
```

Comment:
Well done for only sending back the object instead of an array containing an object 👍